packages/pab/src/spec/pab.edmx

```diff
@@ -0,0 +1,1633 @@
+    </edmx:Reference>
+    <edmx:Reference Uri="https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Validation.V1.xml">
+        <edmx:Include Alias="Validation" Namespace="Org.OData.Validation.V1"/>
+    </edmx:Reference>
+    <edmx:DataServices>
+        <Schema Namespace="UnifiedAiAgentService" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+            <Annotation Term="Core.Links">
+                <Collection>
+                    <Record>
+                        <PropertyValue Property="rel" String="author"/>
+                        <PropertyValue Property="href" String="https://cap.cloud.sap"/>
+                    </Record>
+                </Collection>
+            </Annotation>
+            <EntityContainer Name="EntityContainer">
+                <EntitySet Name="Agents" EntityType="UnifiedAiAgentService.Agents">
+                    <NavigationPropertyBinding Path="chats/history/chat" Target="Agents/chats"/>
+                    <NavigationPropertyBinding Path="chats/history/trace/history" Target="Agents/chats/history"/>
+                    <NavigationPropertyBinding Path="chats/history/trace/tokenConsumption/history/chat" Target="Agents/chats"/>
+                    <NavigationPropertyBinding Path="chats/history/trace/tokenConsumption/history/previous" Target="Agents/chats/history"/>
+                    <NavigationPropertyBinding Path="chats/history/trace/tokenConsumption/history/inputValues/history" Target="Agents/chats/history"/>
+                    <NavigationPropertyBinding Path="chats/history/trace/tokenConsumption/history/group" Target="Agents/chats/history"/>
+                    <NavigationPropertyBinding Path="chats/history/trace/tokenConsumption/trace" Target="Agents/chats/history/trace"/>
+                    <NavigationPropertyBinding Path="chats/history/previous" Target="Agents/chats/history"/>
+                    <NavigationPropertyBinding Path="chats/history/inputValues/history" Target="Agents/chats/history"/>
+                    <NavigationPropertyBinding Path="chats/history/group" Target="Agents/chats/history"/>
+                    <NavigationPropertyBinding Path="chats/agent" Target="Agents"/>
+                    <NavigationPropertyBinding Path="chats/resources/tool" Target="Agents/tools"/>
+                    <NavigationPropertyBinding Path="chats/resources/chat" Target="Agents/chats"/>
+                    <NavigationPropertyBinding Path="tools/agent" Target="Agents"/>
+                    <NavigationPropertyBinding Path="tools/resources/tool" Target="Agents/tools"/>
+                    <NavigationPropertyBinding Path="tools/resources/chat" Target="Agents/chats"/>
+                    <NavigationPropertyBinding Path="tools/config/tool" Target="Agents/tools"/>
+                </EntitySet>
+                <EntitySet Name="AiModels" EntityType="UnifiedAiAgentService.AiModels"/>
+                <ActionImport Name="sendMessage" Action="UnifiedAiAgentService.sendMessage"/>
```

Comment:
This line is added manually. CAP does not add `ActionImport` tag, which is required by our generator. See https://github.com/SAP/cloud-sdk-js/issues/4577#issuecomment-2011683898